scalafmt-tests/src/main/scala/org/scalafmt/util/DiffAssertions.scala

```diff
@@ -1,122 +0,0 @@
-package org.scalafmt.util
-
-import java.io.File
-import java.text.SimpleDateFormat
-import java.util.Date
-import java.util.TimeZone
-
-import org.scalactic.source.Position
-import org.scalatest.exceptions.StackDepthException
-import org.scalatest.exceptions.TestFailedException
-import org.scalatest.funsuite.AnyFunSuiteLike
-
-trait DiffAssertions extends AnyFunSuiteLike {
-
-  import LoggerOps._
-
-  case class DiffFailure(
-      title: String,
-      expected: String,
-      obtained: String,
-      diff: String
-  )(implicit pos: Position)
-      extends TestFailedException(
-        (_: StackDepthException) =>
-          Some(title + "\n" + error2message(obtained, diff)),
-        None: Option[Throwable],
-        pos
-      ) {
-    /* fool Intellij, prepend a stack trace element pointing to the test case
```

Comment:
this thing was very important. we need to achieve the same with `munit` somehow.

Reply:
I will check it out, I was using VS Code and it has a nice functionality where it tries to guess the location of the file, based on the name. Works pretty well, wonder why Intellij does not allow for anything like this.